CMakeLists.txt

```diff
@@ -177,7 +177,13 @@ install(PROGRAMS ${CMAKE_SOURCE_DIR}/bin/docker-entrypoint.sh ${CMAKE_SOURCE_DIR
 install(FILES static_ages.ini
         DESTINATION ".")
 
-option(ENABLE_TESTS "Enable building and running unit tests" ON)
+if(${CMAKE_VERSION} VERSION_LESS 3.11)
+    # The tests use the FetchContent module, which was added in CMake 3.11.
+    set(ENABLE_TESTS_DEFAULT OFF)
+else()
+    set(ENABLE_TESTS_DEFAULT ON)
+endif()
+option(ENABLE_TESTS "Enable building and running unit tests" ${ENABLE_TESTS_DEFAULT})
```

Comment:
```suggestion
include(CMakeDependentOption)
cmake_dependent_option(
    ENABLE_TESTS
    "Enable building and running unit tests"
    ON
    "CMAKE_VERSION VERSION_GREATER_EQUAL 3.11"
    OFF
)
```

Reply:
Rewrote the condition slightly, because `VERSION_GREATER_EQUAL` only exists since CMake 3.7, and the declared minimum is CMake 3.4.